galaxy_ng/tests/integration/conftest.py

```diff
@@ -317,7 +318,14 @@ def ansible_config():
 
 
 def get_ansible_config():
-    return AnsibleConfigFixture
+    if is_sync_testing():
```

Comment:
I don't have a better suggestion, but this feels like it could get confusing pretty quickly.

Maybe stage_test_sync specific utils and/or a conftest.py with implementations of get_ansible_config() that just do the one thing.

Reply:
@chr-stian it seems this may not be needed since the stage tests explicitly call `get_ansible_config_sync` and not `get_ansible_config`?

Reply:
here it's called, and it's a hook that's always called (sync and non sync tests)
https://github.com/chr-stian/galaxy_ng/blob/sync_tests_stage/galaxy_ng/tests/integration/conftest.py#L576